corehq/messaging/scheduling/static/scheduling/js/conditional_alert_list.js

```diff
@@ -120,8 +120,13 @@ hqDefine("scheduling/js/conditional_alert_list", [
                 },
                 success: function (data) {
                     self.showPaginationSpinner(false);
+                    if (_.isString(data)) {
```

Comment:
Feel free to call out of scope, because this isn't part of the logout issue: You could reduce these polls drastically by also checking `_.any(data.rules, function (r) { return r.locked_for_editing; })` - if no rules are being editing, there shouldn't be any progress bar and there's no need to poll.